virtcontainers/qemu.go

```diff
@@ -825,6 +830,22 @@ func (q *qemu) waitSandbox(timeout int) error {
 		return err
 	}
 
+	qemuVersion := fmt.Sprintf("%d.%d.%d", ver.Major, ver.Minor, ver.Micro)
+	if !q.config.UseVSock || version.Compare(qemuVersion, "4.1.0", "<") {
```

Comment:
We already have a small amount of code that checks the qemu version without needing to use an external package. See:

- https://github.com/kata-containers/runtime/blob/master/virtcontainers/qemu_ppc64le.go#L122

I really don't like hard-coding version numbers into programs like this unless *absolutely* essential. I do wonder if this check could be "pushed down" into `govmm` to avoid having it here.

Either way, @markdryan - could you tal at this PR please?

Reply:
@jodh-intel We do have QMP version checks in govmm for certain methods whose underlying QMP API have changed from version to version.  Consequently, these checks cannot be avoided.  

However, in this particular case, we're not calling a method here though, so it's not clear to me how you'd push the version check down to govmm in this case.